test/unit/test-sub-generators-service.js

```diff
@@ -60,6 +60,7 @@ describe('ServiceSubGenerator', () => {
           name: 'a',
           appName: 'b',
           options: {feature: 'c'},
+          testsSeparated: false,
```

Comment:
Could you also change the description of the test? Like add a suffix ` - testsSepared`.